unattended_installer/builder.sh

```diff
@@ -252,6 +255,9 @@ function builder_main() {
     if [ -n "${installer}" ]; then
         buildInstaller
         chmod 500 ${output_script_path}
+        if [ -n "${change_filebeat_url}" ]; then
+            sed -i -E "s|filebeat_wazuh_template=.*|filebeat_wazuh_template=\"https://raw.githubusercontent.com/wazuh/wazuh/\${wazuh_major}/extensions/elasticsearch/7.x/wazuh-template.json\"|g" "${resources_installer}/installVariables.sh"
```

Comment:
This sed should be simplified and less hardcoded

Reply:
Done https://github.com/wazuh/wazuh-packages/pull/2001/commits/e7751d5f2a689477c1f6fcfea11d3d0c28a2467a